src/carcockpit/shaders/shader_adv.cpp

```diff
@@ -107,70 +100,59 @@ shader_adv::shader_adv() :
 						const vec3 Ka = vec3(0.1, 0.1, 0.1);  		   // Ambient reflectivity
 						const vec3 Ks = vec3(0.7, 0.7, 0.7);  		   // Specular reflectivity
 						const float Shininess = 40.0;                  // Specular shininess factor
-					
-						// vec3 phongModelCB( vec3 norm, vec3 diffR, float glossFactor ) 
-						// {
-						// 	vec3 r = reflect( -light_dir, norm );
-						// 	vec3 ambient = light_intensity * Material.Ka;
-						// 	float sDotN = max( dot(light_dir, norm) , 0.0 );
-						// 	vec3 diffuse = light_intensity * diffR * sDotN;
-						// 	vec3 spec = vec3(0.0);
-						// 	if( sDotN > 0.0 )
-						// 		spec = light_intensity * Material.Ks * pow( max( dot(r, view_dir), 0.0 ), glossFactor );
-						// 	return ambient + diffuse + spec;
-						// }
-
-						vec3 phongModel( vec3 norm, vec3 diffR, float glossFactor ) 
+
+						vec3 phongModel( vec3 norm, vec3 diffR, float ambient_occ, float glossiness, float metalness ) // arm = ambient/roughness/metalness
```

Comment:
rename `ambient_occ` -> `ambient_occlusion` ? not much longer, but more descriptive

Reply:
renamed
ambient_occ -> ambient_occlusion
diffR -> diffuse_reflectivity
phongModel -> phong_model